Frameworks/Starboard/EbrFile.cpp

```diff
@@ -163,14 +172,50 @@ bool EbrUnlink(const char* path) {
 
 #define FSROOT "."
 #define mkdir _mkdir
-char g_WritableFolder[2048] = ".";
 
-void EbrSetWritableFolder(const char* folder) {
-    strcpy_s(g_WritableFolder, folder);
+Wrappers::HString _getAppDataPath() {
+    Wrappers::HString toReturn;
+    ComPtr<IStorageFolder> folder;
+    ComPtr<IApplicationDataStatics> applicationDataStatics;
+    ComPtr<IApplicationData> applicationData;
+    ComPtr<IStorageItem> storageItem;
+
+    if (FAILED(GetActivationFactory(Wrappers::HStringReference(RuntimeClass_Windows_Storage_ApplicationData).Get(),
+                                    &applicationDataStatics))) {
+        return toReturn;
+    }
+
+    if (FAILED(applicationDataStatics->get_Current(&applicationData))) {
+        return toReturn;
+    }
```

Comment:
that doesn't exist right now. I suppose I could make it but it seems these are getting out of hand.

---
In reply to: [98271294](https://github.com/Microsoft/WinObjC/pull/1849#discussion_r98271294) [](ancestors = 98271294)